ppr-ui/src/composables/address/BaseAddress.vue

```diff
@@ -191,11 +204,22 @@ export default defineComponent({
       emit('valid', valid)
     }, { immediate: true, deep: true })
 
-    watch(() => country.value, (val) => {
+    watch(() => country.value, (val, oldVal) => {
+      console.log(val)
       if (val === 'CA') {
         schemaLocal.value.postalCode = origPostalCodeRules.concat([baseRules.postalCode])
       } else if (val === 'US') {
         schemaLocal.value.postalCode = origPostalCodeRules.concat([baseRules.zipCode])
+      } else {
+        schemaLocal.value.postalCode = origPostalCodeRules.concat([baseRules.maxLength(15)])
+      }
+      // reset other address fields (check is for loading an existing address)
+      if (oldVal) {
```

Comment:
It might be worth writing a test for this....

Reply:
Yeah good point I will put one in